views/attendance/attendance_edit_class_info.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * enter information to take attendance for new class
+ *
+ * provides a list of fields to specify what class attendance will be recorded for
+ *
+ * @author Scott Hansen
+ * @copyright 2017 Marist College
+ * @version [version number]
+ * @since [initial version number]
+ */
+
+authorizedPage();
```

Comment:
Don't need the `authorizedPage()` function since this is handled in the `page_permissions.ini`.